#! /bin/sh /usr/share/dpatch/dpatch-run
## refocus_automake.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: cleaning up automake weirdness.

@DPATCH@
diff -urNad gimp-plugin-registry~/refocus/src/Makefile.am gimp-plugin-registry/refocus/src/Makefile.am
--- gimp-plugin-registry~/refocus/src/Makefile.am	2003-01-30 22:29:28.000000000 +0100
+++ gimp-plugin-registry/refocus/src/Makefile.am	2007-10-15 16:24:02.000000000 +0200
@@ -7,18 +7,11 @@
 endif
 
 
-SUBDIRS= src lib doc $(GTK_DOC_DIR)
+SUBDIRS= src doc $(GTK_DOC_DIR)
 
 install-bin:
 	cd src; make install-bin
 install-admin-bin:
 	cd src; make install-admin-bin
 
-if HAVE_ATLAS
 dist-hook:
-	@echo "**** Don't run make dist when system is configured with atlas."
-	@echo "**** Rename lib-atlas, rm config.cache and re-run configure."
-	@false
-else
-dist-hook:
-endif
diff -urNad gimp-plugin-registry~/refocus/src/src/Makefile.am gimp-plugin-registry/refocus/src/src/Makefile.am
--- gimp-plugin-registry~/refocus/src/src/Makefile.am	2006-12-18 21:29:22.000000000 +0100
+++ gimp-plugin-registry/refocus/src/src/Makefile.am	2007-10-15 16:24:02.000000000 +0200
@@ -1,22 +1,23 @@
 ## Process this file with automake to produce Makefile.in
 
-INCLUDES=@GTK_CFLAGS@ @GIMP_CFLAGS@ -I @LAPACK_INCLUDE_DIR@
+INCLUDES=
 
 bin_PROGRAMS = refocus
 noinst_PROGRAMS = test-matrix
 
-ldadd_atlas=-L@LAPACK_LIB_DIR@ -llapack -lcblas -latlas
+ldadd_atlas=-llapack -lblas
 
 noinst_HEADERS=bdclosure.h refocus.h matrix.h tilebuf.h conv.h fwlapack.h gimppreview.h prevman.h util.h
 EXTRA_DIST = dummy-plugin.c
 
 refocus_SOURCES = refocus.c conv.c gimppreview.c prevman.c util.c matrix.c tilebuf.c bdclosure.c fwlapack.c
-refocus_LDADD = @GTK_LIBS@ -lgimp-2.0 ${ldadd_atlas}
-refocus_DEPENDENCIES=@LAPACK_LIB_DIR@/liblapack.a
+
+refocus_LDADD = ${ldadd_atlas}
+refocus_DEPENDENCIES=
 
 test_matrix_SOURCES = matrix.c test-matrix.c fwlapack.c
-test_matrix_LDADD = -lm ${ldadd_atlas} @GLIB_LIBS@
-test_matrix_DEPENDENCIES=@LAPACK_LIB_DIR@/liblapack.a
+test_matrix_LDADD = -lm ${ldadd_atlas}
+test_matrix_DEPENDENCIES=
 
 install-bin: refocus
 	@if test "x@GIMPTOOL@" = "x" ; then \
@@ -36,9 +37,6 @@
 		@GIMPTOOL@ --install-admin-bin refocus ; \
 	fi
 
-@LAPACK_LIB_DIR@/liblapack.a:
-	cd @LAPACK_LIB_DIR@; make
-
 %.s: %.c
 	${CC} ${CFLAGS} -S $< -o $@
 
